catalog/files/impl/src/main/java/org/projectnessie/catalog/files/gcs/GcsClients.java

```diff
@@ -148,7 +149,12 @@ static Credentials buildCredentials(
                 oauth2token.expiresAt().map(i -> new Date(i.toEpochMilli())).orElse(null));
         return OAuth2Credentials.create(accessToken);
       default:
-        throw new IllegalArgumentException("Unsupported auth type " + authType);
+        try {
```

Comment:
I'd rather leave the `default` case as it is but add a new enum value (e.g. `WORKLOAD_IDENTITY`) to `GcsBucketOptions.GcsAuthType`.

Reply:
Sure

Reply:
@snazy PTAL